app/assets/stylesheets/components/page_specific/_floating_chat.scss

```diff
@@ -1,13 +1,8 @@
-.chat-box__container {
+.chat-box__container,
+.chat-popup__container {
   background-color: $color-white;
   position: fixed;
-  top:30%;
-  left: 0;
-  padding: $baseline-unit*3;
   z-index:9999;
```

Comment:
z-index is too high please consider a value under 100

Reply:
These styles are outside the scope of this ticket as it refers to the desktop chat button but I can make these changes anyway

Reply:
That's a fair comment @andreferreiradlw but we try to take the approach that if we see a candidate for refactoring whilst working on a file or repo, we make the change. If its a big refactor, you can choose whether to tackle the refactor as part of the ticket or create a new ticket. We do this because some of the code base is quite old and this approach helps to keep it current.

Reply:
@margOnline I totally agree I just missed this one as these styles were already there but I've changed it to 99 now.